lib/shared/utils.js

```diff
@@ -81,3 +82,24 @@ exports.makeFlatStartCaseObject = (object) => {
   });
 
 };
+
+/**
+ * @summary Create a clone of the image, removing .path and adding .filename
+ * @function
+ * @public
+ *
+ * @param {Object} image - image
+ * @returns {Object} image with filename instead of path
+ *
+ * @example
+ * hideImagePath({
+ *   size: 1000000000,
+ *   path: '/home/john/rpi.img'
+ * })
```

Comment:
Maybe we can `console.log` the path property in the example to showcase that the directory name is removed?